docs/docsite/rst/user_guide/playbooks_conditionals.rst

```diff
@@ -201,150 +263,135 @@ to any other tasks within the included file(s). A common situation where this di
     - debug:
         var: x
 
-This expands at include time to the equivalent of::
+Ansible expands this at execution time to the equivalent of::
 
     - set_fact:
         x: foo
       when: x is not defined
+      # this task sets a value for x
+
     - debug:
         var: x
       when: x is not defined
+      # Ansible skips this task, because x is now defined
 
-Thus if ``x`` is initially undefined, the ``debug`` task will be skipped.  By using ``include_tasks`` instead of ``import_tasks``,
-both tasks from ``other_tasks.yml`` will be executed as expected.
+Thus if ``x`` is initially undefined, the ``debug`` task will be skipped. If this is not the behavior you want, use an ``include_*`` statement to apply a condition only to that statement itself.
 
-For more information on the differences between ``include`` v ``import`` see :ref:`playbooks_reuse`.
-
-.. _conditional_imports:
+Starting with Ansible 2.0, you can apply conditions to ``import_playbook`` as well as to the other ``import_*`` statements. When you use this approach, Ansible returns a 'skipped' message for every task on every host that does not match the criteria, creating repetitive output. In many cases the :ref:`group_by module <group_by_module>` can be a more streamlined way to accomplish the same objective; see :ref:`os_variance`.
 
-Conditional Imports
-```````````````````
+.. _conditional_includes:
 
-.. note:: This is an advanced topic that is infrequently used.
+Conditionals with includes
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Sometimes you will want to do certain things differently in a playbook based on certain criteria.
-Having one playbook that works on multiple platforms and OS versions is a good example.
+When you use a conditional on an ``include_*`` statement, the condition is applied only to the include task itself and not to any other tasks within the included file(s). To contrast with the example used for conditionals on imports above, look at the same playbook and tasks file, but using an include instead of an import:
```

Comment:
```suggestion
When you use a conditional on an ``include_*`` statement, the condition is applied only to the include task itself and not to any other tasks within the included file(s). To contrast with the example used for conditionals on imports above, look at the same playbook and tasks file, but using an include instead of an import::
```